arch/arm/boot/dts/meson8m2-minix-neo-x8-h-plus.dts

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+/*
+ * Copyright (c) 2018 Oleg Ivanov <balbes-150@yandex.ru>
+ * Copyright (c) 2018 Martin Blumenstingl <martin.blumenstingl@googlemail.com>
+ * Copyright (c) 2020 Arnaud Mondit <arnaud.mondit@gmail.com>
+ */
+
+/dts-v1/;
+
+#include "meson8m2-mxiii.dtsi"
+
+/ {
+	model = "Minix X8-H Plus";
+	compatible = "minix,x8-h-plus", "amlogic,meson8m2";
+
+	aliases {
+		serial1 = &uart_A;
```

Comment:
this shouldn't be necessary because we're not using this as user-exposed serial port but for communication with the Bluetooth module (so the serial port is handled on the kernel side and doesn't need this alias).

I probably have to clean up some of the existing .dts files which have this though

Reply:
Ok, it was leftover from copypaste from mxiii-plus dts. Removed in last commit